### PR TITLE
validator scores and penalties

### PR DESCRIPTION
**Pull request by jdgemm**
provided content for validator scores and penalties, added link on validator page, and included a sidebar link

### CODE REVIEW

File: docs/mine-hnt/validators/validator-score.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+slug: /mine-hnt/validators/validator-score
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Validator Scores and Penalties
+
+## Validator Scores
+
+Validator nodes earn rewards for work performed while in the consensus group.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx]:
```suggestion
```

File: docs/mine-hnt/validators/validator-score.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Validator nodes start with a base score 1.0. The network uses the score to choose which validator nodes join, and when they are removed from the consensus group.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx]:
```suggestion
Upon joining the Network, validator nodes start with a base score `1.0`. The network uses the score to choose which validator nodes join consensus, and when they are removed from the consensus group.
```

File: docs/mine-hnt/validators/validator-score.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+The score naturally increases when a validator node joins the consensus group and for each round it remains in it.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx]:
```suggestion
After being elected to the consensus group, a validator node's score increases for each round it remains in the group. 
```

File: docs/mine-hnt/validators/validator-score.mdx (hunk 1 above)
Comment (jdgemm) [reply docs/mine-hnt/validators/validator-score.mdx]: what's the change here...

File: docs/mine-hnt/validators/validator-score.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+After the validator node is unelected from the consensus group, over time its score reverts back to 1.0.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx]:
```suggestion
After the validator node is unelected from the consensus group, its score will naturally revert towards 1.0.
```

File: docs/mine-hnt/validators/validator-score.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+| Score | Join or remain in consensus group |
+| --- | :---: |
+| Lower | Higher chance |
+| Higher| Lower chance |
+
+Use of score:
+* Limits the amount of time a validator node stays in the consensus group.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx:28]: I would move this section to the top as the intro paragraph, before the "Validator Scores" section.

File: docs/mine-hnt/validators/validator-score.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+* Prevents a validator node from rejoining the consensus group too quickly.
+
+## Validator Penalites
+
+The consensus group is made up of validator nodes that are rewarded with $HNT for working together to verify transactions and producing blocks.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx:33]: I would remove this entire sentence and start this section with the following paragraph.

File: docs/mine-hnt/validators/validator-score.mdx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+Use of score:
+* Limits the amount of time a validator node stays in the consensus group.
+* Prevents a validator node from rejoining the consensus group too quickly.
+
+## Validator Penalites
+
+The consensus group is made up of validator nodes that are rewarded with $HNT for working together to verify transactions and producing blocks. 
+
+However, if a validator node prevents the consensus group from performing its task, for example, uses insufficient computing resources or a weak network connection (equally strong upstream and downstream is needed), the validator node will incur penalties that can affect the amount of rewards earned.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx:35]:
```suggestion
If a validator node prevents the consensus group from performing its task - for example, uses insufficient computing resources or a weak network connection - the validator node will incur penalties that can affect the amount of rewards earned.
```

File: docs/mine-hnt/validators/validator-score.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+Using the validator scoring mechanism, the network will penalize bad acting validator nodes by increasing their score. As noted above, validator nodes with higher scores will be ejected from the consensus group sooner, and will have a lower probability of joining the consensus group. Over time a validator’s score will revert back to its base score of 1.0.
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/validator-score.mdx:37]:
```suggestion
Using the validator scoring mechanism, the network will penalize bad-acting validator nodes by increasing their score. As noted above, validator nodes with higher scores will be ejected from the consensus group sooner, and will have a lower probability of joining the consensus group. Over time a validator’s score will revert back to its base score of `1.0`.
```

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**